server/routes/api/delete/deleteUserRoutes.ts

```diff
@@ -0,0 +1,75 @@
+import { Router, Request, Response } from "express";
+import { validationResult, param } from "express-validator";
+
+import Handler from "./deleteUserHandler";
+
+const deleteRouter = Router();
+
+const handler = new Handler();
+const BABYSITTER_INVALID_INPUT_ERROR = "babysitterId must be provided and a number";
+const PARENT_INVALID_INPUT_ERROR = "parentId must be provided and a number";
+const ACTIVE_END_TIMESTAMP = '9999999999';
+
+deleteRouter.put("/parent/:parent", 
+    [ 
+        param('parent').notEmpty().isNumeric().withMessage(PARENT_INVALID_INPUT_ERROR)
+    ],
+    async (req: Request, res: Response) => {
+        try {
+            const fieldValidationResult = validationResult(req);
+            if (!fieldValidationResult.isEmpty()) {
+              return res
+                .status(400)
+                .json({ message: fieldValidationResult.array().map((item) => item.msg).join(' ') });
+            }
+            const { parent: parentId } = req.params
+            const data = await handler.getParent(Number(parentId));
+            const userEndTimestamp = data[0]['end_timestamp']
+
+            if (userEndTimestamp != ACTIVE_END_TIMESTAMP) {
+                return res
+                .status(400)
+                .json({ message: 'This user is not active' });
+            }
+            
+            const newEndTimestamp = await handler.deleteParent(Number(parentId));
```

Comment:
I think I won't return the newEndTimestamp. The user doesn't need to know that we keep the updated time by us and not actually delete him from the user.
Therefore - i would just write await handler.deleteParent(Number(parentId)); and not return any data from the route